roles/kube_nginx_ingress/tasks/main.yml

```diff
@@ -8,8 +8,10 @@
       repo: '{{ nginx_ingress_repo }}'
     namespace: '{{ nginx_ingress_namespace }}'
     binary: '{{ bin_dir }}/helm'
-    values:
-    - "{{ lookup('file', role_path ~ '/files/nginx_ingress_values.yml') }}"
+    values: >-
+      {{ [
+        lookup('file', role_path ~ '/files/nginx_ingress_values.yml')
```

Comment:
Would it make sense to add a `|from_yaml` here to detect encoding issues eagerly?

Reply:
Indeed